### ISSUE TITLE
Problem: tablet sized screens fail to render properly

### ISSUE THREAD

**Issue by sjmackenzie** [fractalide/fractalide-website-styx#17]
Problem: Tablet sized screens fail to render properly inthat the black bar at the bottom overlaps the text, though laptap and mobile phones correctly render.

Solution: use CSS to add a new screen size?

**Comment by ericsagnes** [comment]
I just noticed the issue, will add a fix for that into the Japanese translation branch.

**Comment by ericsagnes** [comment]
Note: The main navigation also look funky in tablet sizes.

**Comment by sjmackenzie** [comment]
In that the main nav spreads across the whole screen? I presonally didn't find that strange :-)

**Comment by ericsagnes** [comment]
No, the nav list is too long and drops under the nav brand like in this screenshot:
![navbar-drop](https://user-images.githubusercontent.com/367880/28880075-ddf3e39c-77de-11e7-978b-02445eb9d5ee.png)

**Comment by sjmackenzie** [comment]
Ah I see, this is the case with the English site too, imho with long words like `documentation` and `whitepaper` this might be hard to squeeze into the space next to the logo. Seems alright with it underneath! Maybe just keep it? Up to you how you want it! :-)
